bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/api/ShellyDeviceProfileTest.java

```diff
@@ -163,7 +163,7 @@ private static Stream<Arguments> provideTestCasesForBuildBluServiceName() {
     @Test
     void buildBluServiceNameWhenNameUnknownThrowIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> {
-            ShellyDeviceProfile.buildBluServiceName("sbbt", "001A2B3C4D5E");
+            getBluServiceName("sbbt", "001A2B3C4D5E");
```

Comment:
This test should now be moved to `ShellyDevicesTest` and renamed to `getBluServiceNameWhenNameUnknownThrowIllegalArgumentException`.

Reply:
done